app/assets/stylesheets/components/common/_cookie_message.scss

```diff
@@ -12,24 +12,29 @@
 
 .cookie-message__content {
   @extend %type-small;
-  @include column(11);
+  @include column(12);
   margin-top: $baseline-unit*3;
   margin-bottom: $baseline-unit*3;
+  padding-right: 76px;
```

Comment:
I believe this line should be deleted as your setting padding-right again in line 19

Reply:
Yeah, it's just there for browsers that don't understand `calc()`. Not critical perhaps, but would prefer to specify something in this case.